styles/main.css

```diff
@@ -0,0 +1,651 @@
+  display: grid;
+  align-items: center;
+}
+.text-medium {
+  font-weight: 500;
+}
+.text-regular {
+  font-weight: normal;
+}
+
+.text-bold {
+  font-weight: bold;
+}
+
+.font-s-md {
+  font-size: 15px;
+}
+.font-s-lg {
+  font-size: 20px;
+}
+
+.font-s-xs {
+  font-size: 8px;
+}
+.font-s-sm {
+  font-size: 10px;
+}
+.font-s-rg {
+  font-size: 13px;
+}
+.flex {
+  display: flex;
+}
+
+.rows {
+  flex-direction: row;
+}
+
+.columns {
+  flex-direction: column;
+}
+
+.justify-center {
+  justify-content: center;
+}
+.space-between {
+  justify-content: space-between;
+}
+
+.top {
+  align-self: flex-start;
+}
+.gap-sm {
+  gap: 5px;
+}
+
+.gap-md {
+  gap: 10px;
+}
+
+.gap-lg {
+  gap: 20px;
+}
+
+.center-items {
+  align-items: center;
+}
+
+.vertical-center {
+  vertical-align: middle;
+}
+
+.self-center {
+  align-items: center;
+}
+
+li.end {
+  height: 50px !important;
+  position: fixed;
+  margin-bottom: 30px;
+  align-self: flex-end;
+  bottom: 0;
+}
+
+/**/
+
+.tweet-button {
+  color: white;
+  fill: white;
+  background-color: var(--blue);
+  width: 80%;
+  padding: 10px 0px;
+  box-sizing: content-box;
+  align-items: center;
+  margin: 0 auto;
+  justify-content: center;
+}
+
+.tweet-button-small {
+  color: white;
+  fill: white;
+  background-color: var(--blue);
+  width: 45%;
+  height: 35px;
+  box-sizing: content-box;
+  margin: 0 auto;
+  align-self: flex-end;
+}
+
+.disabled {
+  opacity: 0.5;
+}
+
+.follow-button {
+  background: black;
+  color: white;
+  border: none;
+  height: min-content;
+  align-self: center;
+}
+/* -------------------------------- */
+.img-fit {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  max-width: 100%;
+}
+
+/* AVATARS */
+
+.avatar {
+  height: 50px;
+  width: 50px;
+  position: relative;
+  overflow: hidden;
+  border-radius: 50%;
+  align-self: baseline;
+  justify-self: center;
+}
+
+/* -------------------------------- */
+
+.container {
+  width: 1250px;
+  margin: 0 auto;
+  display: grid;
+  grid-template-columns: min-content 1fr;
+}
+
+li {
+  list-style: none;
+  margin: 20px 0;
+}
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+svg {
+  height: 90%;
+  cursor: pointer;
+}
+
+header {
+  position: sticky;
+  height: 100vh;
+  top: 0;
+}
+
+.nav li a {
+  height: 100%;
+}
+.nav li {
+  height: 35px;
+  vertical-align: middle;
+}
+.nav {
+  position: sticky;
+  top: 25px;
+  width: 250px;
+  height: 100%;
+}
+
+.active h2 {
+  font-weight: bold;
+}
+
+/* MAIN CONTENT */
+
+.content-container {
+  height: 100%;
+  width: 100%;
+  position: relative;
+}
+
+.content {
+  display: grid;
+  grid-template-columns: 600px 1fr;
+  min-height: 100vh;
+}
+
+.content-header {
+  z-index: 1;
+  display: grid;
+  align-items: center;
+  grid-template-columns: 600px 1fr;
+  position: sticky;
+  top: 0;
+  width: 100%;
+  height: 50px;
+}
+
+.main-header {
+  display: grid;
+  grid-template-columns: 1fr 24px;
+  position: relative;
+}
+
+.content-header::before {
```

Comment:
what does ::before do?

Reply:
`before` and `after` add a styling component (pseudo-element) without having to add a new tag in the HTML, it's quite useful for example when you want to add a background that's not exactly in the same position as the element.

[More info](https://developer.mozilla.org/en-US/docs/Web/CSS/::before)